docs/user/guides/build/email-notifications.rst

```diff
@@ -0,0 +1,40 @@
+How to enable email notifications
+=================================
```

Comment:
```suggestion
How to enable build email notifications
=======================================
```

We call them `build notification via email` and `email notifications` below. We should probably call them one specific thing (`build email notifications`?)

Reply:
Same situation on the feature page:

```
Build Notifications and Webhooks
================================

Read the Docs features a number of build notification mechanisms.
Build notifications can alert you when your builds fail so you can take immediate action.

Email notifications:
  Read the Docs allows you to configure emails that can be sent on failing builds.
  This makes sure you know when your builds have failed.

Build Status Webhooks:
  Build notifications can happen via :term:`webhooks <webhook>`.
  This means that we are able to support a wide variety of services that receive notifications.

  Slack and Discord are supported through ready-made templates.

  Webhooks can be customized through your own template and a variety of variable substitutions.
```

I like "build notifications via email" because it allows us to simply say "build notifications" if we need to talk about the broader concept.

Reply:
I can see that "Email notifications" is the most prevailing term. I also think it works fine. I'll try to reword a few places, including the feature page.